tests/fv3_conf/cpld_control_run.IN

```diff
@@ -25,7 +27,11 @@ else
      MOM_IC=@[INPUTDATA_ROOT]/MOM6_IC/${OCNRES}/${SYEAR}${SMONTH}${SDAY}${SHOUR}
      ICE_IC=@[INPUTDATA_ROOT]/CICE_IC/${OCNRES}/${SYEAR}${SMONTH}${SDAY}${SHOUR}
   else
-     FV3_IC=@[INPUTDATA_ROOT]/${FV3_DIR}/INPUT_L127_mx${OCNRES}
+     if [ "$V2_SFC_FILE" = "true" ]; then
+       FV3_IC=/scratch1/NCEPDEV/da/George.Gayno/ufs_utils.git/chgres_cube.fractional/RT/FV3_input_data/INPUT_L127_mx100.v2.sfc
```

Comment:
These paths will need to be reverted once the data is in place and we are ready to start testing this PR.

Reply:
If you can place the data for the new tests in the official location, I can update my branch to point to it.

Reply:
I renamed the new input directories. Can you adjust like FV3_IC=@[INPUTDATA_ROOT]/${FV3_DIR}/INPUT_L127_mx${OCNRES}_v2_sfc ?

Reply:
Done at 9f9f8f2.

Reply:
And de655ec.